src/applications/hca/helpers.jsx

```diff
@@ -498,6 +498,63 @@ export const isEssentialAcaCoverageDescription = (
     </div>
   </div>
 );
+export const healthInsuranceCoverageQuestionDescription = (
+  <div className="vads-u-margin-bottom--3">
+    <div className="hca-tooltip-wrapper">
+      <AdditionalInfo triggerText="Why we ask this information">
```

Comment:
We should be using `<va-additional-info trigger="...">` instead

Reply:
changed as indicated, ty